app/services/trainees/create_from_hesa.rb

```diff
@@ -107,14 +107,18 @@ def course_attributes
         course_min_age: age_range && age_range[0],
         course_max_age: course_max_age,
         study_mode: study_mode,
-        itt_start_date: hesa_trainee[:itt_start_date],
+        itt_start_date: trainee_start_date,
```

Comment:
I think this reads a little oddly.

Should the base calculation be called something other than `trainee_start_date`?

Reply:
how about... `def itt_start_date`? 🙃

Reply:
that kinda does make more sense hah. `trainee_start_date: itt_start_date` means the trainee started on time.

Reply:
Ha, yeah that way round makes more sense I think. Could also have a comment that we can't distinguish between the two from HESA, hence setting the trainee start date to the ITT start date.